setup/cocci.m4

```diff
@@ -285,7 +285,7 @@ AC_DEFUN([AC_COCCI_TOOL],
     AC_SUBST([$1], [no])
   ],
   [dnl  find the tool
-    AC_COCCI_FINDTOOL([$1],[[$]$1])
+    AC_COCCI_FINDTOOL([$1],[$2])
```

Comment:
Has '[$]' a special meaning? Looks like a typo. In any case, the second arg should probably passed.